File: Algoritmo(C)/Alg_S7_1.c
#include <stdio.h>
int main(){
    //variables
    int n, maior = 0;
    //input
    printf("Digite um numero: ");
    scanf("%d",&n);
    //process
    while (n != 0){
        if (n > maior){
            maior = n;
        }
        printf("Digite um numero: ");
        scanf("%d",&n);
    }
    printf("O maior numero e: %d",maior);
}

File: Algoritmo(Python)/Alg_S7_1.py
#variables
maior = 0
#input
print("0 - Finaliza")
N = int(input("Digite um número: "))
#process
while(N != 0):
    if(N > maior):
        maior = N
    N = int(input("Digite um número: "))
print("O maior número é: {0}".format(maior))